Improve browsing experience (#1401)
meilisearch/documentation: resources/faq.md

---
permalink: /faq.html
sidebar: auto
---

# FAQ

> This FAQ is still a work in progress.

Author: Good point, let's do it.

> This FAQ is still a work in progress.

Author: ```suggestion
This FAQ is created in collaboration with our users.
```

> If you have any questions, we want to hear from you! Your feedback will help us improve this page.

Author: ```suggestion
If you have any questions that aren't answered here, we want to hear from you! Your feedback will help us improve our documentation.
```
